### DISCUSSION TITLE
Enterprise version

### DISCUSSION THREAD

**Discussion by Fwhirlwind** [HewlettPackard/swarm-learning#171] [Q&A]
Hello，I am a graduate student interested in using swarm learning for my research projects. I have been exploring the Community version of swarm learning available on Github, but I am interested in learning more about the Enterprise version of swarm learning.

Could you please advise me on how to obtain the Enterprise version of swarm learning for research purposes? Are there any specific requirements or criteria that I need to meet in order to access it? Additionally, are there any resources or documentation available that can help me better understand the features and capabilities of the enterprise edition?

Thank you in advance for your assistance.

**Comment by iArpanPatel** [comment]
As you have already reached out on [Slack](https://hpedev.slack.com/archives/C04A5DK9TUK) for the same, we are closing this discussion.
